src/main/resources/application.yaml

```
@@ -8,11 +8,16 @@ spring:
       max-file-size: 10MB
       max-request-size: 10MB
   batch:
+
+    job:
+      enabled: true
     jdbc:
       initialize-schema: never
+
     job:
```

YAMLlint (1.37.1)

[error] duplication of key "job" in mapping

(key-duplicates)